main.c

```diff
@@ -15,11 +15,18 @@
 #define WHITE_LED_OFF (LATC6 = 1)
 #define BLUE_LED_ON (LATC7 = 0)
 #define BLUE_LED_OFF (LATC7 = 1)
+#define GET_BATTERY_VOLTAGE ADCC_GetSingleConversion(RA0)
+#define GET_CURRENT_DRAW ADCC_GetSingleConversion(RA1)
+#define GET_BATTERY_VOLTAGE ADCC_GetSingleConversion(RB5)
```

Comment:
It looks like RB5 and RA1 are never explicitly initialized for use with the ADC.

Reply:
So I also need to initialize the specific registers for the pins I'm using right?

Reply:
That's correct. If you assign those pins as ADC pins within MCC directly, it'll take care of it for you.